### ISSUE TITLE
Postman Interceptor never connects (MacOS Mojave)

### ISSUE THREAD

**Issue by DJ-Icebear** [a85/POSTMan-Chrome-Extension#8932]
**Describe the bug**
Postman Interceptor is never connecting to my Chrome extension (on MacOS Mojave).

**To Reproduce**
I do a fresh app install
I install the chrome extension
I install the bridge using the app, but it never connects.

**Screenshots**
![postman_interceptor](https://user-images.githubusercontent.com/7745142/90149309-a3b21500-dd84-11ea-8cac-68addff9562d.gif)

**App information (please complete the following information):**
Native Postman App version 7.30.1
MacOS 10.14.6 (Mojave)

**Additional context**
Tried: Restarting computer and reinstalling the postman app and the chrome extension. No matter what I do the status always remains: "INTERCEPTOR DISCONNECTED"

The Postman App DevTools gives me this output:
[-1597330212007.log](https://github.com/postmanlabs/postman-app-support/files/5069747/-1597330212007.log)

**Comment by DannyDainton** [comment]
Hey @DJ-Icebear 

Thanks for writing in, are you able to also try restarting the Browser?

https://support.postman.com/hc/en-us/articles/360025849993-Why-is-the-Postman-Interceptor-Disconnected-

There are also a set of troubleshooting steps listed here on the Learning Center:

https://learning.postman.com/docs/sending-requests/capturing-request-data/interceptor/#troubleshooting-tips

Please also doubly confirm that you're installing this extension, lot's of imposters out there 😉

https://chrome.google.com/webstore/detail/postman-interceptor/aicmkgpgakddgnaphhhpliifpcfhicfo?hl=en

**Comment by DJ-Icebear** [comment]
Hello! Thanks for a quick response!

The troubleshooting tips led me in the right direction and I managed to solve my issue!

When I navigated to `/Users/myuser/.postman/InterceptorBridge` and ran the executable there, I could see an error:
```
dyld: Library not loaded: /usr/local/opt/icu4c/lib/libicui18n.64.dylib
  Referenced from: /usr/local/bin/node
  Reason: image not found
Abort trap: 6
```
Which let me on the trail of something being wrong with my node installation.
After doing: `brew upgrade node` and `brew link --overwrite node`
I can run the executable without errors, and get the connected status working! 🎉
